### ISSUE TITLE
Some func_detail creating portals/splits

### ISSUE THREAD

**Issue by hemebond** [ericwa/ericw-tools#338]
Some func_detail brushes are affecting the portals as if they were structural

```
---- qbsp / ericw-tools v0.18.1-148-gfea7849-dirty ----
---- vis / ericw-tools v0.18.1-148-gfea7849-dirty ----
---- light / ericw-tools v0.18.1-389-g982ef527 ----
```

Detail brushes hidden
![image](https://user-images.githubusercontent.com/424218/205318240-2da22d36-922b-4714-9ce7-49743537da59.png)

Detail brushes visible
![image](https://user-images.githubusercontent.com/424218/205318335-e270fddc-e2e1-4e14-a385-6c3853ed6c3a.png)

**Comment by Paril** [comment]
Do you have a .map or anything we can try to repro with? Without your specific setup screenshots aren't much help.

**Comment by hemebond** [comment]
I've just pushed the [current version of the map](https://github.com/hemebond/quakemash/blob/e1c533073e95a0676056dc0dc6fad6d558b2b587/maps/src/quakemash.map) to my repo. This specific example is from the `e3m5` layer.

**Comment by ericwa** [comment]
For reference, the camera in those screenshots is at around `-9896 12232 -7496`.

IIRC I did repro this with the map when the issue was posted, need to check whether it still repros in the current 2.0.0-alpha6.

**Comment by ericwa** [comment]
Seems to be fixed on current `brushbsp` branch (b12894de64c86912712cb8c1aef22a033d6027e0).

Detail visible in TB:

![image](https://github.com/ericwa/ericw-tools/assets/239161/6c2c7758-13cc-4e36-8a0a-f7320e742fa5)

Detail hidden in TB:

![image](https://github.com/ericwa/ericw-tools/assets/239161/65dbe390-f97d-4550-a6ca-5f0826569221)

Note, in the slightly older 2.0.0-alpha6 build, there are more splits in this area coming from the water.
I just fix that yesterday in 765d95e189db775f4750018dfc1d84334460ec36.

Also, for this test, I marked all layers of the map "omit from export" except for e3m5.
